mypyc/test-data/irbuild-frozenset.test

```diff
@@ -113,3 +113,237 @@ L0:
     r5 = r4 >= 0 :: signed
     r6 = truncate r4: i32 to builtins.bool
     return r6
+
+[case testFrozenSetFromRange1]
```

Comment:
`frozenset(range(...))` seems very rare. Do you have evidence that it's a bottleneck in some real-world codebase? If not, it would be sufficient to add a run test (as part of an existing run test to avoid slowing down the test suite), since irbuild tests require more maintenance.

Reply:
I've removed these ones from the PR.